articles/differences-between-artificial-intelligence-machine-learning-and-deep-learning-explained/index.md

```diff
@@ -0,0 +1,120 @@
+### Introduction
+In 1956, [John McCarthy,](/https://en.wikipedia.org/wiki/John_McCarthy_(computer_scientist)/) a professor of mathematics and a group of scientists set out for an ambitious two-months project to create computers that could form abstractions, process language, improve themselves and solve all kinds of problems reserved for humans. Though they were never able to solve the biggest part of the AI equation, this ambitious project kick started the field that has come to be known as Artificial Intelligence (AI) today.
```

Comment:
This sentence is actually very long. It can provide difficulty when reading as one might lose the train of thought reading it. And there are several more that are also long. You can use [Hemingway app](http://www.hemingwayapp.com/) to help you identify the long sentences.